src/SearchResults.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Table } from "reactstrap";
+import moment from "moment";
+
+const SearchResults = props => {
+  return (
+    <Table className="col-12">
+      <thead>
+        <tr className="col-12">
+          <th className="mx-auto bg-danger text-white">Title</th>
+          <th className="mx-auto bg-dark text-white">First Name</th>
+          <th className="mx-auto bg-danger text-white pl-2">Last Name</th>
+          <th className="mx-auto bg-dark text-white pr-3">Email</th>
+          <th className="mx-auto bg-danger text-white pl-1 pr-5">Room id</th>
+          <th className="mx-auto bg-dark text-white pl-2">Check in date</th>
+          <th className="mx-auto bg-danger text-white pl-2">Check out date</th>
+          <th className="mx-auto bg-dark text-white pl-2">Nights</th>
+        </tr>
+      </thead>
+      <tbody>
+        {props.results.map((element, index) => {
+          return (
+            <tr key={index} className="col-12">
+              <th scope="row" className="bg-dark text-white">
```

Comment:
Completely optional idea to make this even tidier: Can you make it so that you don't need to manually alternate the `bg-danger` and `bg-dark` classes? (And so that you don't need to put `text-white` on each cell)?

Hint: Look up the `first-child` and `nth-child` selectors

Reply:
I didn't understand it daniel, how can I fix it?